src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
```

Comment:
Don't fix container size (if there is no such a requirement). Instead, consider using padding and allow the content to dictate the size. If the fixed size is necessary, ensure it's justified by the design requirements.